software_steering_council.md

```diff
@@ -0,0 +1,52 @@
+# Software Steering Council
+
+## Purpose
+
+The Jupyter Software Steering Council (SSC) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn't explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy.
+
+## Responsibilities
+
+### Sole responsibilities
+
+The SSC is responsible for the following:
+
+- Defining the submission, review, and approval process for Jupyter Enhancement Proposals.
+- Stewarding the JEP process to ensure that it is inclusive and participatory, and solicits feedback from the right stakeholders. As a starting point, [the NumFOCUS DISCOVER Cookbook](https://github.com/numfocus/DISCOVER-Cookbook) may offer some useful starting pointers.
```

Comment:
We (@fperez @afshin @ellisonbg) envision the JEP process being used for a broader range of cross-project matters than standards. The primary purpose of the SSC is to make cross-project software decisions with JEPs as their primary mechanism. The SSC could delegate certain aspects of the management of the process to other groups or individuals at their discretion – in these governance documents we are attempting to give the SSC a scope and mandate (JEPs) and leave it up to them to figure out how that works.

Reply:
I think of it in terms of skill set. The people that oversee the JEP process will need to have training and experience in a particular kind of participatory decision making process, so that they can effectively steward and improve the JEP process itself. I don’t think the entire SSC would have these skills, and as such it may be productive to designate specific group of people that oversee the JEP process. The standards group feels like it must also have this skill set (since creating standards means wrangling lots of people to make a decision) so seems like a natural home to steward the JEP process itself